far/FarRus.hlf.m4

```diff
@@ -200,7 +200,7 @@ $ #Far: параметры командной строки#
 
  #-p[<path>]#
  Искать плагины в папке, указанной в <path>.
- Можно указать несколько путей поиска, разделённых символом ';'. При указании
+ Можно указать несколько путей поиска, разделённых символом ‹;›. При указании
```

Comment:
Why exactly [single guillemets](https://codepoints.net/U+2039?lang=en)?
According to [Wikipedia](https://en.wikipedia.org/wiki/Quotation_mark#Summary_table) they are widely used in many languages, but not in Russian.

(I'm not an expert, just curious)

Reply:
It was a bit voluntary. Since English uses “” double quotes for phrases and FarEng.hlf uses single quotes for single characters, and Russian uses «» for phrases, I decided to “fill the gap” and use the ‹› single quotes for single characters in FarRus.hlf. If it causes trouble, we can replace them by any other characters using a regexp replace.

Reply:
It was a bit voluntary. Since English uses “” double quotes for phrases and FarEng.hlf uses single quotes for single characters, and Russian uses «» for phrases, I decided to “fill the gap” and use the ‹› single quotes for single characters in FarRus.hlf. If it causes trouble, we can replace them by any other characters using a regexp replace.

Reply:
Should be fine, as long as the usage is consistent.